skafka/src/main/scala/com/evolutiongaming/skafka/producer/ProducerLogging.scala

```diff
@@ -2,18 +2,22 @@ package com.evolutiongaming.skafka.producer
 
 import cats.data.{NonEmptyMap => Nem}
 import cats.implicits._
-import com.evolutiongaming.catshelper.{Log, MonadThrowable}
+import cats.MonadThrow
+import com.evolutiongaming.catshelper.Log
 import com.evolutiongaming.skafka.{OffsetAndMetadata, ToBytes, Topic, TopicPartition}
 import com.evolutiongaming.smetrics.MeasureDuration
+import org.apache.kafka.common.errors.RecordTooLargeException
 
 object ProducerLogging {
 
   private sealed abstract class WithLogging
 
-  def apply[F[_]: MonadThrowable: MeasureDuration](producer: Producer[F], log: Log[F]): Producer[F] = {
+  /**
+    * @param charsToTrim a number of chars from record's value to log when producing fails because of a too large record
+    */
+  def apply[F[_]: MonadThrow: MeasureDuration](producer: Producer[F], log: Log[F], charsToTrim: Int = 1024): Producer[F] = {
```

Comment:
now seeing this like that - I think we should still keep an old method, otherwise it would be binary compatible :(

Reply:
Sure, I will add one more method to preserve the compatibility